docs/getting-started/flashing-firmware/nrf52/update-techo-booloader.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: update-techo-bootloader
+title: How to Update the LilyGo T-Echo Bootloader to the Latest Version
+sidebar_label: Update T-Echo Bootloader
+sidebar_position: 5
+---
+
+If you're experiencing issues with updating or flashing newer versions of the Meshtastic firmware, and your LilyGo T-Echo is not running the latest bootloader version (0.6.1), updating the bootloader may resolve these problems.
+
+To check which version of the bootloader your device is running, place the device into DFU mode by double-pressing the reset button. Then, open the mounted drive that appears on your computer and check the INFO_UF2.TXT file.
+
+Below are the steps to update your bootloader.
```

Comment:
I guess this could be the level 2, `## Updating bootloader` and then the ones below it could be h3's